HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -277,6 +286,109 @@ def self.get_values_from_air_infiltration_measurements(hpxml_bldg, cfa, weather)
     return { sla: sla, ach50: ach50, nach: nach, volume: infil_volume, height: infil_height, a_ext: a_ext }
   end
 
+  # Calculate ACH50 for annual energy simulation when only leakiness description is provided
+  #
+  # @param hpxml_bldg [HPXML::Building] HPXML Building object representing an individual dwelling unit
+  # @return [Float] Calculated ACH50 value based on IECC zone, cfa, infiltration height, year built, foundation type, ducts and leakiness description
+  def self.calc_ach50(cfa, ncfl_ag, year_built, ceil_height, infil_volume, iecc_cz, foundations, ducts, leakiness_description)
```

Comment:
Need to describe all the arguments, no longer `hpxml_bldg`.

Also, we need an `air_sealed` argument (default to false for OS-HPXML) if we are eventually going to call this from the HEScore code. And it will need to be incorporated it into the calculations below.